config/locales/admin.en.yml

```diff
@@ -98,6 +98,7 @@ en:
       date: Date
       day_total: Daily total
       days: Days
+      times: Sent mails
```

Comment:
the `times` key is not descriptive, can you change it to `sent_emails` or similar?